src/main/java/net/dries007/tfc/world/classic/worldgen/WorldGenLooseRocks.java

```diff
@@ -68,20 +68,9 @@ public void generate(Random random, int chunkX, int chunkZ, World world, IChunkG
                 {
                     veins.removeIf(v -> {
                         if (!v.type.hasLooseRocks()) return true;
-
```

Comment:
If we're going to remove the full scan check for ore blocks, we should introduce a better check for vein proximity:
 - In 1.7.10, it was pointed out to me that the maximum depth scanned was much smaller than we have here, it should probably be adjusted to match. We can do a check of the "center" of the vein, and use the vertical size + vein size modifier to get an approximate value for the highest ore block.
 - The horizontal detection is (at the moment) chunk restricted. Ideally, the veins will still use position / distances to check validity, and the chunk data ore only becomes a sanity check that any ores actually generated.

Reply:
To pin this down, i'm commenting the line i found in classic:
https://github.com/TerraFirmaCraft/TFC-1.7.10/blob/9979960a7c13c19e4e6d018aa774c8a707c85592/src/Common/com/bioxx/tfc/WorldGen/Generators/WorldGenLooseRocks.java#L61